examples/tests/sdk_tests/bucket/copy.test.w

```diff
@@ -0,0 +1,32 @@
+bring cloud;
+bring util;
+
+let b = new cloud.Bucket();
+
+test "copy()" {
+  let KEY1 = "file1.main.w";
+  let VALUE1 = "bring cloud;";
+  let KEY2 = "file2.txt";
+  let VALUE2 = { msg: "Hello world!" };
+
+  b.put(KEY1, VALUE1);
+  b.putJson(KEY2, VALUE2);
+  let file1SrcMetadata = b.metadata(KEY1);
+  let file2SrcMetadata = b.metadata(KEY2);
+
+  // Sleep 100ms to ensure 'metadata.lastModified' changes upon copy.
+  util.sleep(100ms);
+
+  b.copy(KEY1, KEY1);
+  b.copy(KEY2, "dir/${KEY2}");
+  let file1DstMetadata = b.metadata(KEY1);
+  let file2DstMetadata = b.metadata("dir/${KEY2}");
+
```

Comment:
can you please check that the value under the new key is equal to the value under the new key? (in key2 case)